plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintPlugin.kt

```diff
@@ -156,18 +155,10 @@ open class KtlintPlugin : Plugin<Project> {
              */
             fun getPluginConfigureAction(): (Plugin<Any>) -> Unit = {
                 target.extensions.configure(BaseExtension::class.java) { ext ->
-                    ext.sourceSets { sourceSet ->
-                        sourceSet.all { androidSourceSet ->
-                            // Passing Callable, so returned FileCollection, will lazy evaluate it
-                            // only when task will need it.
-                            // Solves the problem of having additional source dirs in
-                            // current AndroidSourceSet, that are not available on eager
-                            // evaluation.
-                            createTasks(
-                                androidSourceSet.name,
-                                target.files(Callable { androidSourceSet.java.srcDirs })
-                            )
-                        }
+                    ext.variants.all { variant ->
```

Comment:
This `ext.variants` is coming from the `PluginUtils`. 

Here `all` makes it lazy loop over variants. When a variant is created, it is sourceSet is also set and contains all directories.